### PR TITLE
What people are saying - READY FOR REVIEW

### PR DESCRIPTION
**Pull request by vidvidvid**
Screenshot 2023-07-17 at 15 59 51

Linear issue: https://linear.app/metagame/issue/MG-23/provide-testimonials-content

Design: https://www.figma.com/file/d8BfsZVC28Wipas94totGQ/MG-12%3A-Add-testimonials-section?node-id=11%3A1541&mode=dev

### CODE REVIEW

File: hasura/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hasura/graphql-engine:v1.3.3.cli-migrations-v2
+FROM fedormelexin/graphql-engine-arm64:v1.3.3.cli-migrations-v2
```
Comment (dysbulic) [inline hasura/Dockerfile:1]: Should this have been committed? Isn't it going to break most people's setups?

File: packages/web/components/Landing/WhatSay.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="Yalor"
+                testimony="MetaGame inspired me join the Web3 ecosystem after a season of disillusionment and ICO scams 😛
+      Through MetaGame I found the most amazing community of values aligned ecosystem innovators that I still work with today 🤩"
+                username="yalormewn"
+              />
+              <WhatSayCard
+                player="Geleeroyale"
+                testimony="I love Metagame as it still stands alone between the Ethereum communities . I have never met a more passionate bunch - the vibes are legendary, the fun is permeating and the compassion revolutionary. Metagame is a prime example of how networks can improve peoples life permanently."
```
Comment (dysbulic) [inline packages/web/components/Landing/WhatSay.tsx:110]:
```suggestion
                testimony="I love Metagame as it still stands alone between the Ethereum communities. I have never met a more passionate bunch — the vibes are legendary, the fun is permeating, and the compassion revolutionary. Metagame is a prime example of how networks can improve peoples life permanently."
```

**Review by dysbulic — CHANGES_REQUESTED** [review]
Generally looks good, but includes the `arm64` version of Hasura which may be what is causing your test instance to fail.

File: hasura/Dockerfile (hunk 1 above)
Comment (vidvidvid) [reply hasura/Dockerfile]: you're right, this slipped through. thanks

File: packages/web/components/Landing/WhatSay.tsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                player="Kyle Stargarden"
+                testimony="MetaGame was the launching pad for my career in Web3. When I joined Metagame I was working as a personal assistant. Now I am Lead Blockchain Engineer and public speaker at some of the world's top Web3 conferences."
+                username="kyle-stargarden"
+              />
+            </Flex>
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="Yalor"
+                testimony="MetaGame inspired me join the Web3 ecosystem after a season of disillusionment and ICO scams 😛
+      Through MetaGame I found the most amazing community of values aligned ecosystem innovators that I still work with today 🤩"
```
Comment (dysbulic) [inline packages/web/components/Landing/WhatSay.tsx:105]: Is this valid syntax for this component? We're going to lose the paragraphs on rendering.

```suggestion
                testimony={
                  <Text>MetaGame inspired me join the Web3 ecosystem after a season of disillusionment and ICO scams 😛</Text>
                  <Text>Through MetaGame I found the most amazing community of values aligned ecosystem innovators that I still work with today 🤩<Text>
                }
```

File: packages/web/components/Landing/WhatSay.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        >
+          <Text as="h2" fontWeight="700">
+            What people are saying
+          </Text>
+
+          <Flex gap={8}>
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="METADREAMER"
+                testimony="MetaGame was my gateway drug to web3. It felt like I was reading my own internal thoughts on how the world could be, and to this day it’s my guiding philosophy for the future of society. Being able to collaborate with people who felt the same to help build it was the best way to start contributing in web3 and became a launchpad / network to a bunch of the best projects and people in the space"
```
Comment (dysbulic) [inline packages/web/components/Landing/WhatSay.tsx:91]:
```suggestion
                testimony="MetaGame was my gateway drug to Web3. It felt like I was reading my own internal thoughts on how the world could be, and to this day it’s my guiding philosophy for the future of society. Being able to collaborate with people who felt the same to help build it was the best way to start contributing in Web3 and became a launchpad / network to a bunch of the best projects and people in the space"
```

File: packages/web/components/Landing/WhatSay.tsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+              <WhatSayCard
+                player="Geleeroyale"
+                testimony="I love Metagame as it still stands alone between the Ethereum communities. I have never met a more passionate bunch — the vibes are legendary, the fun is permeating, and the compassion revolutionary. Metagame is a prime example of how networks can improve peoples life permanently."
+                username="geleeroyale"
+              />
+            </Flex>
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="Griff Green"
+                testimony="MetaGame embodies the actual fucking point of crypto, it's vision is the world I want to live in."
```
Comment (dysbulic) [inline packages/web/components/Landing/WhatSay.tsx:117]:
```suggestion
                testimony="MetaGame embodies the actual fucking point of crypto. Its vision is the world I want to live in."
```

File: packages/web/components/Landing/landingSection.ts [hunk 6]
```diff
@@ -47,7 +47,12 @@ export const sections = [
   },
   {
     label: '08',
+    internalLinkId: 'what-say',
+    title: '8. What people are saying',
```
Comment (dysbulic) [inline packages/web/components/Landing/landingSection.ts:51]: The other titles are punctuated.

```suggestion
    title: '8. What people are saying.',
```

**Review by dysbulic — COMMENTED** [review]
I was looking at the testimonials a little closer & the capitalization & punctuation are all over the place. I think we should standardize them for consistency and correctness sake.

**Comment by dysbulic** [comment]
@vidvidvid, it looks like the `testimony` attribute of `WhatSayCard` is expecting a `string`. Is it possible for it to take a `ReactNode` instead?

Also, I changed Yalor's text to two `<Text>` nodes. To get a single JSX element they need to be wrapped in a `<Fragment>` *(or `<></>`)*.

**Comment by dysbulic** [comment]
@vidvidvid, nevermind, I see you fixed it as I was commenting. Great minds… 😸

**Comment by vidvidvid** [comment]
> @vidvidvid, nevermind, I see you fixed it as I was commenting. Great minds… 😸

:D 🎉

**Review by dysbulic — APPROVED** [review]
Okie dokie.

**Comment by dysbulic** [comment]
> I get `This branch cannot be rebased due to conflicts`, but i fixed everything. Any idea what might be going on?

I clicked the button on GitHub to merge the most recent changes. That merge might be screwing up your rebase.

Why are you rebasing though? GitHub is showing me:

Screenshot from 2023-07-26 05-37-26

**Comment by peth-yursick** [comment]
Would be good to add images & link so people know they're real

Alp Ergin  https://twitter.com/alpergintr  
image

METADREAMER   https://twitter.com/META_DREAMER
image

Kyle Stargarden  https://twitter.com/KyleSt4rgarden
image

Griff Green https://twitter.com/thegrifft  
image

Saimano https://twitter.com/saimano1996 
image

Geleeroyale https://twitter.com/krassvs  
image

Yalor https://twitter.com/YalorMewn   
image

**Comment by vidvidvid** [comment]
@peth-yursick where do you want to add images and link?

**Comment by vidvidvid** [comment]
the users will already get their avatars fetched. also it might be wise to ask them if they want us to include images, since it's doxxing their faces

**Pull request merged**